lib/fog/azurerm/requests/compute/delete_snapshot.rb

```diff
@@ -0,0 +1,33 @@
+
+module Fog
+  module Compute
+    class AzureRM
+      # Real class for Compute Request
+      class Real
+        def delete_snapshot(resource_group_name, snapshot_name, async = false)
```

Comment:
If this request is called asynchronously, then the response return should be as done [here](https://github.com/fog/fog-azure-rm/blob/master/lib/fog/azurerm/requests/compute/delete_virtual_machine.rb#L6)